netci.groovy

```diff
@@ -1459,7 +1484,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                             Utilities.addArchival(newJob, "stdout.txt")
                             Utilities.addArchival(newJob, "Logs/**")
                         }
-                        else if (architecture == 'x64' || architecture == 'x86') {
+                        else if (architecture == 'x64' || architecture == 'x86' || architecture == 'x86lb') {
                             buildCommands += "tests\\runtest.cmd ${runtestArguments}"
```

Comment:
runtest.cmd is most likely not going to understand x86lb as a architecture. There used to be some logic either to use a created TestEnv or use a checked in env.

At a minumum the test env will need to define:

COMPlus_AltJit=*
COMPlus_AltJitName=legacyjit